### PR TITLE
[stable/prometheus]: remove unnecessary clusterrolebindings for alert…

### PR DESCRIPTION
**Pull request by eviln1**
**What this PR does / why we need it**:
This PR removes unneeded `clusterRoleBindings` for `node-exporter` and `alertmanager` components.
Since they do not communicate with the kubernetes API, they do not need any kind of RBAC.
The `clusterRoleBindings` that were created, however, gave the `cluster-admin` role to `node-exporter` and `alertmanager` service accounts.

See https://github.com/kubernetes/charts/pull/1443#pullrequestreview-56912581

### CODE REVIEW

**Comment by eviln1** [comment]
Rebased and resolved conflicts after https://github.com/kubernetes/charts/pull/4397 was merged.

**Comment by unguiculus** [comment]
Would you mind updating this to match our current RBAC best practices?

https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md

File: stable/prometheus/values.yaml [hunk 1]
```diff
@@ -1,15 +1,27 @@
 rbac:
   create: true
 
+## Define serviceAccount names for components. Defaults to component's fully qualified name.
+##
+serviceAccounts:
+  alertmanager:
+    create: false
```
Comment (unguiculus) [inline stable/prometheus/values.yaml]: Set all defaults to `true`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by eviln1** [comment]
@unguiculus all should be good now.

**Comment by eviln1** [comment]
@mgoodness any chance of your blessing ?

**Review by unguiculus — CHANGES_REQUESTED** [review]
There should be a manifest for each of the service accounts you configured. The conditions to create them must be based on the respective `create` flag, not `rbac.create` which is only for RBAC resources, not service accounts.

The readme must be updated regarding service accounts.

**Comment by eviln1** [comment]
@unguiculus Hopefully this is suitable.

**Comment by unguiculus** [comment]
Next time please merge instead of rebasing, so Github can show your changes. Thanks.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**